### DISCUSSION TITLE
Help with HxInputDate , doesn't open

### DISCUSSION THREAD

**Discussion by Guiprirod** [havit/Havit.Blazor#784] [Q&A] [answered]
Hello everyone, somebody know why my HxInputDate doesn't open?

This is my class:
![image](https://github.com/havit/Havit.Blazor/assets/105490638/37081c2d-fcce-4ea4-9014-b17f5e7d8c17)

This is my code:
![image](https://github.com/havit/Havit.Blazor/assets/105490638/7a762e4c-ba35-42be-882a-a4859e5d0593)

When I click in my HxInputDate, nothing deploy , I don't see any error in the console
![image](https://github.com/havit/Havit.Blazor/assets/105490638/5de4670e-98c2-4b03-ba2b-e1ed6b282970)

**Comment by hakenr** [comment]
Hello,
unfortunately, I don't have enough details to assess the cause.
* What render mode do you use? (HxInputDate is an **interactive** component.)
* Do you have Bootstrap JavaScript included in your page? (usually `App.razor`)
* When you click the icon, is there any message displayed in browser console?

**Reply by Guiprirod** [reply]
Hello Robert, thank you for your time.

Answering you:
   - **About render**: I add this render,  but doesn`t work to.
![image](https://github.com/havit/Havit.Blazor/assets/105490638/41858f64-1815-4225-a17c-cc5674e7c663)
    - **About Bootstrap**, I show you my App.razor (think is working)
![image](https://github.com/havit/Havit.Blazor/assets/105490638/0c2608fb-9e3e-4b28-ba98-6ff65a3ff6fe)
     - About message: No, when I click it, I don't see any message in browser console. This is a capture after I clicked the HxInputCalendar several times:
  
![image](https://github.com/havit/Havit.Blazor/assets/105490638/e0360a8b-f048-4329-b43d-9c20b1f7c809)

**Reply by Guiprirod** [reply]
I am attaching my repository if you want to go deeper.

Thank you very much for your help!

https://github.com/Guiprirod/EnglishAcademyProject

**Comment by hakenr** [comment]
@Guiprirod I will take a look at your repo, but commit&push your latest changes. The current repo state does not have the `Form.razor` content mentioned above.

**Reply by Guiprirod** [reply]
@hakenr Sorry, I forgot it, I pushed my changes,
thanks you!

**Comment by hakenr** [comment, marked as answer]
I created a pull-request in your repo which fixes the issues.
Just needed a little consolidation of the styles & JS references in `App.razor`:
https://github.com/Guiprirod/EnglishAcademyProject/pull/1

**Reply by Guiprirod** [reply]
Thank you very much Robert, works correctly!! 😄

**Comment by Guiprirod** [comment]
If anybody have the same issue and need help,
Summary from @hakenr:

In my case, I imported wrongly the App.razor.

**Before the help this is my App.razor** 

![image](https://github.com/havit/Havit.Blazor/assets/105490638/e9cda8ac-1ee4-4086-b167-2e7cbf1c2333)


After @hakenr helped me

![image](https://github.com/havit/Havit.Blazor/assets/105490638/cd6e58fd-2a5b-4e96-abda-1368c6ed6c88)

Thank you!
